nifi-nar-bundles/nifi-aws-bundle/nifi-aws-abstract-processors/pom.xml

```diff
@@ -130,6 +146,11 @@
             <groupId>com.github.ben-manes.caffeine</groupId>
             <artifactId>caffeine</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+            <version>2.15.3</version>
```

Comment:
This version should not be set explicitly. It should be provided in the `jackson-bom` at the root Maven configuration, did you notice any issues with the version not set?

Reply:
No, I can remove the explicit version here.